src/commands/run/jobs-builder/deploy-job-builder.ts

```diff
@@ -36,6 +36,31 @@ export class DeployJobBuilder {
           parameters: this.mapProjectParams(project, params),
         },
       }));
+
+    return this.mergeDoubledProjects(mappedProjects);
+  }
+
+  private mergeDoubledProjects(projects: JobDescriptor[]) {
+    const filteredProjects = new Map<string, JobDescriptor>();
+
+    projects.forEach(project => {
+      const currentProject = filteredProjects.get(project.opts.name);
+
+      if (currentProject) {
+        Object.keys(project.opts.parameters).forEach(key => {
+          if (
+            project.opts.parameters[key] !== currentProject.opts.parameters[key] &&
```

Comment:
How about creating two variables to store `project.ops.paremeters` and `currentProject.opt.parameters`? Now it's a little bit unclear. What do you think?

Reply:
Well, I disagree. We're using not only `project.ops.parameters` but also `project.displayName`. Creating variables for parameters only will still require to use the original object and I'm not a fan of that mix. I can create 4 new variables but that's overkill I guess. Unless you have other, specific proposition? Or maybe renaming the existing variables would help?

Reply:
Okay, I agree.